src/main/java/com/automatedtest/sample/utils/Link.java

```diff
@@ -0,0 +1,14 @@
+package com.automatedtest.sample.utils;
```

Comment:
this is not an utils, you should create a new package for it, like `control` or sth